ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_BestaetigungsrelevanteSysteme.md

```diff
@@ -46,4 +46,8 @@ ISiK-Modul Terminplanung [z.B. für die Umsetzung von KHZG Fördertatbestand 2]
 - KIS
 - Patientenportal-Terminmodul
 
-Hinweise und Rückfragen zu den bestätigungsrelevante Systemen können an die [DKG-Geschäftsstelle](https://www.dkgev.de/dkg/verband-auf-einen-blick/geschaeftsstelle/) gerichtet werden.
\ No newline at end of file
+Hinweise und Rückfragen zu den bestätigungsrelevante Systemen können an die [DKG-Geschäftsstelle](https://www.dkgev.de/dkg/verband-auf-einen-blick/geschaeftsstelle/) gerichtet werden.
+
+## Weitere Festlegungen auf Profilebene im CapabilityStatement
+
+Festlegungen zu Anforderungen auf Profilebene werden im [ISIKCapabilityStatement](https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/ImplementationGuide-markdown-CapabilityStatement?version=current) geführt.
```

Comment:
funktionieren eigentlich auch relative Markdown-Links?

Reply:
gute frage! @alexzautke , @ylboerner geht das? wenn nicht, wäre das ein neues Feature in Simplifier wert.